.github/workflows/ci.yml

```diff
@@ -75,7 +75,9 @@ jobs:
         run: npm run check
       - name: Install extra dependencies for building Git on Ubuntu
         if: matrix.targetPlatform == 'ubuntu'
-        run: sudo apt-get install libcurl4-openssl-dev libexpat1-dev gettext
+        run: |
+          sudo apt-get update
+          sudo apt-get install libcurl4-openssl-dev libexpat1-dev gettext
```

Comment:
This was needed to prevent this 404 error: https://github.com/actions/virtual-environments/issues/3989#issuecomment-909401936